subsys/net/lib/Kconfig

```diff
@@ -6,6 +6,8 @@
 
 menu "Network Protocols"
 
+source "subsys/net/lib/sockets/Kconfig"
```

Comment:
I am fine with having a separate directory for this so we do not clutter subsys/net/ip too much.